### DISCUSSION TITLE
Rendering Font Icons without Anti-Aliasing

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#10406] [Q&A]
I created a OTF font file containing a few icons. This allows the icon color to match the font color, disabled state, theme, etc. There's just no better alternative.

One thing I'm noting, though is anti-aliasing on the icon. Stop button is a square, with anti-aliased edges instead of square edges. Is there a way to disable anti-aliasing? Either in Avalonia rendering, or in FontForge settings?

**Comment by Gillibald** [comment]
https://github.com/AvaloniaUI/Avalonia/pull/9584

**Reply by mysteryx93** [reply]
Ah, brand new feature that hasn't been merged yet! OK that can wait.
